charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+# Docs Working Group Charter (Proposed)
+
+## Mission and Goals
+
+The Docs Working Group serves as a support, helper, and advisory body to Jupyter subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on docs across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on docs in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## The Docs Working Group's Activities
+
+The Docs Working Group will focus on the efforts described below, in service of its mission:
+
+- Help write docs (inside the bounds of each subproject's governance)
+    - This includes meta docs, cross-cutting docs (items relevant to multiple subprojects), developer and contributor docs, non user-facing docs and others
+- Develop recommendations and guidance:
+	- For communicating common information and concepts across the Jupyter ecosystem
+        - Because we want to encourage consistency across the Jupyter ecosystem and between different projects
+	- For style and best practices when authoring documentation media
+        - Including technical advice, such as tools-usage and markdown guidance that helps support high level goals, like interlinking/cross-connection between subprojects
+- Community engagement
+	- To help connect users with information, expertise and resources
+	- To gather feedback from the community about what needs to be documented/what information they need
+- Help support and improve all aspects of documentation across the Jupyter ecosystem
+	- This means, for members of the group, also taking on any related (non-document) tasks that will be beneficial to docs and support the Docs Working Group's goals, such as:
+        - Developing application extensions related to docs, like in-app integrated docs, guided in-application tours or hoverable "what's this" features
+        - Automations for gathering ReadTheDocs traffic stats to gauge user interest
```

Comment:
Some of these seem a bit too technical and specific to be included in the charter, I'd suggest being more high level and tool agnostic.

Reply:
How about this @blink1073 ?

> - This means, for members of the group, also taking on any related (non-document) tasks that will be beneficial to docs and support the Docs Working Group's goals, such as:
>     - Improve the user experience in Jupyter applications through offline or interactive documentation;
>     - Implement analytics for (ReadTheDocs) documentation webpages for better understanding of docs use.

Reply:
I think the important part here is "This means...also taking on any related (non-document) tasks", to explicitly call out that our scope includes development work and other non-writing work that supports the mission. Most of these items are intended to be clarifying examples that illustrate the intended meaning. I do take your point about being tool-agnostic though... @chbrandt I think that looks good.